pkg/config/setup/config.go

```diff
@@ -1086,6 +1086,11 @@ func InitConfig(config pkgconfigmodel.Config) {
 	// The histogram buckets use to track the time in nanoseconds it takes for a DogStatsD listeners to push data to the server
 	config.BindEnvAndSetDefault("telemetry.dogstatsd.listeners_channel_latency_buckets", []string{})
 
+	// Agent Telemetry
+	// Enable various telemetry on the internals of the Agent.
+	// This will not tirgger any billable custom metrics.
```

Comment:
Let's remove this mention for now, can be re-introduced once the feature is not experimental anymore

Reply:
Agree, fixed in the last commit